server/controllers/conversation.js

```diff
@@ -0,0 +1,33 @@
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+
+// Given two userIDs, create a conversation
+exports.createConversation = asyncHandler(async (req, res) => {
+    const { senderID, receiverID } = req.body;
+    const newConvo = new Conversation({
+        participants: [senderID, receiverID],
+    });
+
+    try {
+        const savedConvo = await newConvo.save();
+        res.status(200).json(savedConvo);
+    } catch (error) {
+        res.status(500).json(error);
+    }
+})
+
+//Based on userID in the params, get all conversations of the user
+
+exports.getConversations = asyncHandler(async (req, res) => {
+    const { userID } = req.params;
```

Comment:
Please protect your routes using the `auth` middleware. 
Once done, you can access the user's id in the request object.